io.github.baldeuniversel.am-okay.yml

```diff
@@ -0,0 +1,48 @@
+id: io.github.baldeuniversel.am-okay
+runtime: org.freedesktop.Platform
+runtime-version: '23.08'
+sdk: org.freedesktop.Sdk 
+
+command: am-okay
+
+finish-args:
+    - --filesystem=host
+    - --filesystem=home 
+    - --filesystem=/tmp
+    - --device=all
```

Comment:
Does it really need device access? I don't think [this permission](https://docs.flathub.org/docs/for-app-authors/requirements#device-access) does what you think it does.

Reply:
Yep, I know .
A few users have asked me to do this.

There is another scenarios to achieve this goal (I removed the entry `--device`) .